modules/virt-aws-bm.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * virt/install/preparing-cluster-for-virt.adoc
+
+:_content-type: CONCEPT
+[id="virt-aws-bm_{context}"]
+= {VirtProductName} on AWS bare metal
+
+You can run {VirtProductName} on an Amazon Web Services (AWS) bare metal {product-title} cluster. Before you set up your cluster, review the following summary of supported features and limitations:
+
+Installing::
+
+* You can install the cluster by using installer-provisioned infrastructure, ensuring that you specify bare metal node instance types in the `install-config.yaml` file. For example, you can use the `c5n.metal` type value for a machine based on x86_64 architecture.
+** For more information, see the {product-title} documentation about installing on AWS.
+
+Accessing virtual machines (VMs)::
+
+* There is no change to how you access VMs by using the `virtctl` CLI tool or the {product-title} web console.
+* You can expose VMs by using a `NodePort` or `LoadBalancer` service. 
+** The load balancer approach is preferable because {product-title} automatically creates the load balancer in AWS and manages its lifecycle. A security group is also created for the load balancer, and you can use annotations to attach existing security groups. When you remove the service, {product-title} removes the load balancer and its associated resources.
+
+Networking::
+
+* Because connecting a secondary network to a VM is not supported on AWS, you must use the primary network, which you can expose by using services. 
+* You cannot use Single Root I/O Virtualization (SR-IOV) or bridge Container Network Interface (CNI) networks, including virtual LAN (VLAN). If your application requires a flat layer 2 network or control over the IP pool, consider using OVN-Kubernetes secondary overlay networks.
+
+Storage::
+--
+* Amazon Elastic File System (EFS) and Amazon Elastic Block Store (EBS) are not supported for use with {VirtProductName}.
+* Red Hat tests the following storage providers on AWS: link:https://docs.portworx.com/install-portworx/openshift/operator/[Portworx] and {rh-storage-first}.
```

Comment:
@aglitke Do we support CNV on AWS with ODF ?

@jpeimer Did storage QE test CNV on AWS with ODF ?

At-least virt QE has never tested CNV on AWS with ODF, We ( Virt QE)  have never tested with ODF and I do not know, if LiveMigration works with ODF StorageClass on AWS.

Virt QE only tested with Portworx StorageClass.